source/MainWindow.cpp

```diff
@@ -234,6 +237,24 @@ MainWindow::~MainWindow()
 	delete fUndoContext;
 }
 
+void
+MainWindow::AboutRequested()
+{
+	BAboutWindow* about = new BAboutWindow("ResourceEdit","x-vnd.Haiku-ResourceEdit");
+	
+	const char* authors[] = {
+		"Tri-Edge AI",
+		"waddlesplash",
+		"Panagiotis Vasilopoulos",
+		"Artur Jamro",
+		NULL
+	};
+
+	about->AddCopyright(2012-2013, "Tri-Edge AI");
```

Comment:
"2012-2013" = -1 :)
Just add the start year, i.e. 2012.

Reply:
Should've seen that one coming, my bad.